Maestro/subscriptions.json

```diff
@@ -748,8 +746,8 @@
         "https://github.com/dotnet/efcore/blob/release/3.1//**/*",
         "https://github.com/dotnet/efcore/blob/release/5.0//**/*",
         "https://github.com/dotnet/efcore/blob/release/6.0//**/*",
-        "https://github.com/dotnet/razor-compiler/blob/release/6.0.2xx//**/*",
-        "https://github.com/dotnet/razor-tooling/blob/release/3.1//**/*"
+        "https://github.com/dotnet/razor/blob/release/6.0.2xx//**/*",
```

Comment:
Think this should stay razor-compiler. We're not merging the old release branches

Reply:
Thanks, reverted this line.